_config/view.yml

```diff
@@ -0,0 +1,6 @@
+---
+Name: view-config
+---
+SilverStripe\Core\Injector\Injector:
+  SilverStripe\View\TemplateEngine:
+    class: 'SilverStripe\View\SSTemplateEngine'
```

Comment:
Explicitly set this one by default.
We're not moving the template engine into its own module yet, that'll be the next set of PRs - but I'm on the fence about whether this config would sit here or in that module. I'm thinking here, but if you have opinions either way I'd be glad to hear them.